tasks.py

```diff
@@ -225,25 +233,27 @@ def build(_c, debug=False):
     for target in constants.TARGETS:
         first_image = TARGETS[target]["tags"][0]
 
-        LOG.info("Building %s...", first_image)
+        LOG.info("Building %s (platform %s)...", first_image, PLATFORM)
         try:
             image = CLIENT.images.build(
                 path=str(CWD),
                 target=target,
                 rm=True,
                 tag=first_image,
+                platform=PLATFORM,
```

Comment:
This does not seem to be effective.  I believe it is related to https://github.com/docker/docker-py/issues/2822

Reply:
This was merged and a beta release was cut.  Just waiting on GA now I think, will test as soon as I have time